src/rust/src/expr/string.rs

```diff
@@ -1,4 +1,4 @@
-use crate::{prelude::*, PlRExpr, RPolarsErr};
+use crate::{datatypes::PlRDataType, prelude::*, PlRExpr, RPolarsErr};
```

Comment:
Please remove the prefix and run formatter.

```suggestion
use crate::{PlRDataType, prelude::*, PlRExpr, RPolarsErr};
```

Reply:
Done